### PR TITLE
Update necessary configurations to ensure the React app runs locally

### PR DESCRIPTION
**Pull request by Sy36613**
Create login and sign up page

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import React, { useState,useEffect } from 'react';
 import { Routes, Route, Navigate } from 'react-router-dom';
-import GptsListPage from './pages/GptsListPage';
+import GptsListPage from './pages/gptsListPage';
```
Comment (MooyGhul) [inline client/src/App.tsx:3]: Why is this changed?

**Review by MooyGhul — APPROVED** [review]

File: client/src/App.tsx (hunk 1 above)
Comment (Sy36613) [reply client/src/App.tsx:3]: Fixed the typo when import 'gptsListPage' file
